tests/torch/test_layer_attributes.py

```diff
@@ -246,7 +246,7 @@ def __str__(self):
     ),
     LayerAttributesTestDesc(
         module=nn.EmbeddingBag(1, 1),
-        model_input_info_list=[ModelInputInfo([1, 1], type_str='long')],
+        model_input_info_list=[ModelInputInfo([1, 1], type_str='long', filler='zeros')],
```

Comment:
Seems like more strict verification were added to `EmbeddingBag`
With ones filler it started to fail with error:
`
ret, _, _, _ = torch.embedding_bag(
>           op1 = operator(*args, **kwargs)
E           RuntimeError: Index 0 is out of bounds: 1, range 0 to 1
`
Indexes should be zeros.